test/metabase/api/persist_test.clj

```diff
@@ -43,3 +45,39 @@
         (is (= default-cron
                (get-in (task.persist-refresh/job-info-by-db-id)
                        [(:id db) :schedule])))))))
+
+(deftest persisted-info-by-id-test
+  (with-setup db
+    (t2.with-temp/with-temp
```

Comment:
```suggestion
    (mt/with-temp
```

mt/with-temp is basically `t2.with-temp/with-temp` at the moment, but we have a linter to force us to use the prior version for consistency.

Reply:
Thanks, that makes sense to use `mt/with-temp`.

Committed in https://github.com/metabase/metabase/pull/33776/commits/8321ffdb27db00be8c309d7e54cd186592ecf0e6